src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -474,6 +485,49 @@ internal void AddAsset(List<AssetData> assets, string assetName, string version,
             });
         }
 
+        internal (List<PackageArtifactModel>,
+            List<BlobArtifactModel>) MergeManifests(List<Manifest> parsedManifest)
+        {
+            List<PackageArtifactModel> packageArtifacts = new List<PackageArtifactModel>();
+            List<BlobArtifactModel> blobArtifacts = new List<BlobArtifactModel>();
+
+            for (int i = 0; i < parsedManifest.Count; i++)
+            {
+                Manifest manifest = parsedManifest[i];
+                foreach (var package in manifest.Packages)
+                {
+                    PackageArtifactModel packageArtifact = new PackageArtifactModel()
+                    {
+                        Attributes = new Dictionary<string, string>
```

Comment:
[nit] The braces here do not seem to be aligned properly.

Reply:
outdated code